js/personal.js

```diff
@@ -94,19 +106,42 @@ View.prototype = {
             OC.Notification.showTemporary(t('facerecognition', 'There was an error trying to show your friends'));
         });
     },
+    setEnabledUser: function (enabled) {
+        var self = this;
+        $.ajax({
+            type: 'GET',
+            url: OC.generateUrl('apps/facerecognition/setuservalue'),
+            data: {
+                'type': 'enabled',
+                'value': enabled
+            },
+            success: function () {
+                if (enabled) {
+                    OC.Notification.showTemporary(t('facerecognition', 'The analysis is enabled, please be patient, you will soon see your friends here.'));
+                } else {
+                    OC.Notification.showTemporary(t('facerecognition', 'The analysis is disabled. In the next execution your data will be completely eliminated.'));
```

Comment:
consider rewording `your data will be completely eliminated` to something less scary, for example `The analysis is disabled. We will remove data about all your recognized faces.` (this way, it seems like some user data might be removed, and "eliminated" seems like something that Daleks from Dr. Who would say:D)

Reply:
or, as you said in other place, "You will lose all the information analyzed"